package.json

```diff
@@ -42,7 +43,7 @@
     "five-bells-condition": "^3.2.0",
     "five-bells-shared": "^17.0.0",
     "keypair": "^1.0.0",
-    "knex": "^0.10.0",
+    "knex": "^0.11.9",
```

Comment:
The older version used a different name for the hostname in the config for mssql only